MAUI/DataGrid/Unbound-Column.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: Unbound Columns in MAUI DataGrid control | Syncfusion
+description: Learn here all about unbound columns support in Syncfusion MAUI DataGrid (SfDataGrid) control and more about it.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Unbound column in MAUI DataGrid (SfDataGrid)
+
+The data grid allows adding additional columns that are not bound with data objects from the underlying data source.The unbound column can be added using the [SfDataGrid.DataGridUnboundColumn]() class.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfDataGrid 
+            x:Name="datagrid"    
+            ColumnWidthMode="Fill"
+            AutoGenerateColumnsMode="None"
+            ItemsSource="{Binding OrderInfoCollection}">
+    <syncfusion:SfDataGrid.Columns>
+         <syncfusion:DataGridUnboundColumn 
+                        MappingName="DiscountPrice"
+                        HeaderText="SUM"
+                        Expression="Price1+Price2"
+                        Format="C" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDataGrid dataGrid = new SfDataGrid();
+            OrderInforRepo orderInforRepo = new OrderInforRepo();
+            dataGrid.ItemsSource = orderInforRepo;
+            DataGridUnboundColumn DiscountColumn = new DataGridUnboundColumn()
+            {
+                MappingName = "DiscountPrice",
+                HeaderText = "SUM",
+                Expression = "Price1+Price2",
+                Format = "C"
+            };
+            this.datagrid.Columns.Add(DiscountColumn);
+            
+
+{% endhighlight %}
+{% endtabs %}
+
+
+<img alt="Unbound-column" src="Images\unbound-column\maui-datagrid-unbound-column.png" width="404" />
```

Comment:
Alt text should be "Unbound column in MAUI DataGrid".

Reply:
changed the alt